packages/sdk-demo/src/components/Layout/ThemeSelector.tsx

```diff
@@ -0,0 +1,114 @@
+import React, { useRef, useState } from 'react';
+
+import { ThemeConfig } from '@/types';
+import type { I18n } from '@lingui/core';
+import { t } from '@lingui/macro';
+import { useLingui } from '@lingui/react';
+import { useRootElements } from '@monite/sdk-react';
+import DarkModeIcon from '@mui/icons-material/DarkMode';
+import KeyboardArrowDownIcon from '@mui/icons-material/KeyboardArrowDown';
+import KeyboardArrowUpIcon from '@mui/icons-material/KeyboardArrowUp';
+import LightModeIcon from '@mui/icons-material/LightMode';
+import {
+  Button,
+  Typography,
+  Menu,
+  MenuItem,
+  Divider,
+  FormControlLabel,
+  Switch,
+} from '@mui/material';
+
+const getButtonLabel = (themeIndex: string, i18n: I18n) => {
+  switch (themeIndex) {
+    case 'material':
+      return t(i18n)`Material UI`;
+    case 'monite':
+      return t(i18n)`Monite`;
+    default:
+      return t(i18n)`Material UI`;
+  }
+};
+
+interface ThemeSelectorProps {
+  themeConfig: ThemeConfig;
+  onChange: (themeConfig: ThemeConfig) => void;
+}
+
+export const ThemeSelector = ({
+  themeConfig,
+  onChange,
+}: ThemeSelectorProps) => {
+  const { i18n } = useLingui();
+  const { root } = useRootElements();
+
+  const [isMenuOpen, setIsMenuOpen] = useState(false);
+  const buttonRef = useRef<HTMLButtonElement>(null);
+
+  const { themeIndex, colorMode } = themeConfig;
+
+  return (
+    <>
+      <Button
+        ref={buttonRef}
+        id="theme-selector"
+        sx={{ justifyContent: 'space-between' }}
+        aria-controls={isMenuOpen ? 'theme-menu' : undefined}
+        startIcon={colorMode === 'dark' ? <DarkModeIcon /> : <LightModeIcon />}
+        endIcon={
+          isMenuOpen ? <KeyboardArrowUpIcon /> : <KeyboardArrowDownIcon />
+        }
+        aria-haspopup="true"
+        aria-expanded={isMenuOpen ? 'true' : undefined}
+        aria-label="theme-menu-button"
+        variant="outlined"
+        onClick={() => setIsMenuOpen(!isMenuOpen)}
+      >
+        {getButtonLabel(themeIndex, i18n)}
+        <Menu
+          id="theme-selector"
```

Comment:
Probably, `useId(...)` result should be used instead

Reply:
fixed